### PR TITLE
Adds a placeholder starting points menu

### LINKED ISSUE #147: create "Starting Points" component as placeholder
Create a component "StartingPoints" which is the left nav "Create Resources" part of the existing BFE.  In our case, it's just a placeholder, or maybe there's a link to "monograph instance" only and a "select a resource template" sort of button.

I'm just making that up, as it's meant as a placeholder, with a "Starting Points" heading visible.

### PR DESCRIPTION
**Pull request by jgreben**
Fixes #147

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -31,6 +31,7 @@ body {
   font-size: 13.5px;
 }
 #footer {
+  float: right;
```
Comment (jermnelson) [inline src/styles/main.css]: Is there reason why the footer is floating right? I wasn't able to see any difference in the appearance of the footer when this css was removed and when it was present.

Comment (jgreben) [reply src/styles/main.css]: It has to do with when the browser window is made narrower, otherwise it bumps to the right side of the page.

Comment (jgreben) [reply src/styles/main.css]: @jermnelson see comment above, but it does looks like there is some further CSS refinement that needs to happen. Not sure if this is the PR for that given that we will be doing work on the form later on.

File: src/components/editor/Editor.jsx [hunk 2]
```diff
@@ -118,16 +119,25 @@ class Editor extends Component {
...
+    let startingPoints = {
+      border: '1px dotted',
+      float: 'left',
+      padding: '20px'
+    }
     return(
       <div id="editor">
         <EditorHeader triggerEditorMenu={this.props.triggerHandleOffsetMenu}/>
         <h1> Editor Page </h1>
         <p>(will have a "Choose Resource Template" area + editing area)</p>
```
Comment (ndushay) [inline src/components/editor/Editor.jsx]: don't need this line anymore, correct?

File: src/components/editor/ResourceTemplate.jsx [hunk 3]
```diff
@@ -12,22 +12,27 @@ class ResourceTemplate extends Component {
...
-          <li>resourceURI: <strong>{this.props.resourceTemplates[0].resourceURI}</strong></li>
-          <li>id: <strong>{this.props.resourceTemplates[0].id}</strong></li>
-          <li>remark: <strong>{this.props.resourceTemplates[0].remark}</strong></li>
-        </ul>
-        <form id="resourceTemplate" className="form-horizontal" role="form" style={dottedBorder}>
-          <h4>BEGINNING OF FORM</h4>
-          <PropertiesWrapper propertyTemplates = {[this.props.resourceTemplates[0].propertyTemplates]} />
-          <h4>END OF FORM</h4>
-        </form>
+      <div>
```
Comment (ndushay) [inline src/components/editor/ResourceTemplate.jsx]: why do we need another div?

File: src/components/editor/StartingPoints.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React, { Component }  from 'react'
+
+class StartingPoints extends Component {
+  render() {
+    return (
+      <div>
```
Comment (ndushay) [inline src/components/editor/StartingPoints.jsx]: no identifying class or id for this div?

File: src/styles/main.css (hunk 1 above)
Comment (SaravShah) [reply src/styles/main.css]: hmmm @jgreben i'm not seeing the problem when the browser is made narrower. Could you show me whenever you have time?

Comment (SaravShah) [reply src/styles/main.css]: ^saw when webpage was enbiggened or however you spell it

File: src/components/editor/Editor.jsx (hunk 2 above)
Comment (jgreben) [reply src/components/editor/Editor.jsx]: 👍

File: src/styles/main.css (hunk 1 above)
Comment (jgreben) [reply src/styles/main.css]: 👌

File: src/components/editor/ResourceTemplate.jsx (hunk 3 above)
Comment (jgreben) [reply src/components/editor/ResourceTemplate.jsx]: 🔨 Fixed.

**Pull request merged**